docs/src/content/docs/resources/themes.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Themes
+description: Style your docs with a community theme for Starlight
+tableOfContents: false
+sidebar:
+  order: 1
+head:
+  - tag: style
+    content: |
+      body { --sl-content-width: 66.75rem; }
+---
+
+import ThemeGrid from '~/components/theme-grid.astro';
+
+A theme is a Starlight plugin that changes the visual appearance of a site with [custom CSS](/guides/css-and-tailwind/), [component overrides](/guides/overriding-components/), or other new features.
+
+## Community Themes
+
+Install a theme built by the community to quickly customize the look and feel of your site.
+
+<ThemeGrid
+	labels={{
+		/** Accessible label for the theme toggle. */
+		toggle: 'Preview theme',
+		/** Accessible label for the dark color scheme variant. */
+		dark: 'Dark',
+		/** Accessible label for the light color scheme variant. */
+		light: 'Light',
+	}}
+	themes={[
+		{
+			title: 'Starlight Rapide',
+			description:
+				'Starlight theme inspired by the Visual Studio Code Vitesse theme.',
+			href: 'https://starlight-theme-rapide.vercel.app/',
+			previews: { light: 'rapide-light.png', dark: 'rapide-dark.png' },
+		},
+		{
+			title: 'Starlight Obsidian Theme',
+			description:
+				'Starlight theme inspired by the style of Obsidian Publish sites.',
+			href: 'https://fevol.github.io/starlight-theme-obsidian/',
+			previews: { light: 'obsidian-light.png', dark: 'obsidian-dark.png' },
+		},
+		{
+			title: 'Catppuccin for Starlight',
+			description: 'Soothing pastel theme for Starlight.',
+			href: 'https://catppuccin-starlight.otterlord.dev/',
+			previews: { light: 'catppuccin-light.png', dark: 'catppuccin-dark.png' },
+		},
+		{
+			title: 'Ion',
+			description: 'A sleek, modern theme for Starlight.',
+			href: 'https://louisescher.github.io/starlight-ion-theme/',
+			previews: { light: 'ion-light.png', dark: 'ion-dark.png' },
+		},
+		{
+			title: 'Starlight Black',
+			description: 'Starlight theme inspired by shadcn docs.',
+			href: 'https://starlight-theme-black.vercel.app/',
+			previews: { light: 'black-light.png', dark: 'black-dark.png' },
+		},
+		{
+			title: 'Starlight Flexoki',
+			description:
+				'A warm and friendly theme based on the Flexoki color palette.',
+			href: 'https://delucis.github.io/starlight-theme-flexoki/',
+			previews: { light: 'flexoki-light.png', dark: 'flexoki-dark.png' },
+		},
+	]}
+/>
+
+:::tip[Add your own!]
+Have you built a Starlight theme?
+[Open a PR](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md#themes) adding your theme to this page!
+:::
```

Comment:
In all resource pages, this block is always at the top of the page, is it expected to be at the bottom here?

Reply:
Mostly because with the wide layout I find it a bit distracting at the top? (As I side note: I think I’d be tempted to move them all down as I’m not sure they’re the most high priority content on these pages?)

Reply:
I guess the CTA at the top was more relevant before when these pages were brand new, not a lot of plugins/content, etc. but now I think they would be fine all at the bottom too?